packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -2070,16 +2070,11 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
         (event: CustomEvent<DhType.ViewportData>) => {
           try {
             const { detail: data } = event;
-            if (data.rows.length === 0) {
-              // Table is empty, wait for the next updated event
-              return;
+            let values: unknown[] = [];
+            if (data.rows.length > 0) {
+              const row = data.rows[0];
+              values = keyTable.columns.map(column => row.get(column));
             }
```

Comment:
What are we changing this section for? It purposely returned when `data.rows.length === 0` before so it didn't do an update before a row had ticked in.